schema/v1.1.0/common.yaml

```diff
@@ -755,6 +919,17 @@ enums:
         description: Annotations were generated automatically.
       hybrid:
         description: Annotations were generated semi-automatically.
+  annotation_file_shape_type_enum:
+    description: Describes the shape of the annotation
+    permissible_values:
+      SegmentationMask:
+        description: A binary mask volume
+      OrientedPoint:
+        description: A series of coordinates and an orientation
+      Point:
+        description: A series of coordinates
+      InstanceSegmentation:
+        description: A volume with labels for multiple instances
 
```

Comment:
Should we have SemanticSegmentationMask as an entry here?

Reply:
I don't think so! We support `SemanticSegmentationMask` as a `source` type in the ingestion tooling, but that gets written out as a standard `SegmentationMask` after ingestion:

https://github.com/chanzuckerberg/cryoet-data-portal-backend/blob/main/ingestion_tools/scripts/importers/annotation.py#L280-L282